src/lib/annotations/drawing/DrawingThread.js

```diff
@@ -96,55 +112,65 @@ class DrawingThread extends AnnotationThread {
     //--------------------------------------------------------------------------
 
     /**
-     * Set the drawing styles
+     * Set the drawing styles for a provided context. Sets the context of the memory context if
+     * no other context is provided.
      *
      * @protected
      * @param {Object} config - The configuration Object
      * @param {number} config.scale - The document scale
      * @param {string} config.color - The brush color
+     * @param {CanvasContext} [otherContext] - Optional context to be set
```

Comment:
The otherContext was meant to be any other provided context to be drawn to. In the case of this PR, the other context was intended to be the commited drawings context.

In any case, this will be done away with if SVGs are used in place of canvas drawings. For now, otherContext will be renamed to context (ambiguity intended as it should be able to style any context provided)